src/product-type-export.js

```diff
@@ -246,11 +247,21 @@ export default class ProductTypeImport {
     })
   }
 
+  addFilterCondition (client) {
+    const keys = this.config.productTypeFilter.replace(/,/g, '","')
+    return client.where(`key IN ("${keys}")`)
```

Comment:
I am not sure we need to pass the `client` object to this method.

Let's make the method generate the keys and return it.